legend-engine-xt-serviceStore-executionPlan/src/main/java/org/finos/legend/engine/plan/execution/stores/service/IServiceStoreExecutionExtension.java

```diff
@@ -33,8 +35,14 @@ static List<IServiceStoreExecutionExtension> getExtensions()
         return Lists.mutable.withAll(ServiceLoader.load(IServiceStoreExecutionExtension.class));
     }
 
-    default List<Function3<SecurityScheme, HttpClientBuilder, MutableList<CommonProfile>, Boolean>> getExtraSecuritySchemeProcessors()
+    default List<Function5<SecurityScheme, AuthenticationSpecification, Credential, RequestBuilder, HttpClientBuilder, Boolean>> getExtraSecuritySchemeProcessors()
```

Comment:
Do we need AuthenticationSpecification here?

Reply:
We dont need it.  Removed.